delivery_package_number/models/stock_picking.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    number_of_packages = fields.Integer(
+        string="Number of Packages",
+        store=True,
+        readonly=False,
```

Comment:
This is not valid for v12, so you need to regular field + onchanges/writes.

Reply:
I don't think this works properly in UI, as this is not supported till v13. I'm talking about onchange + write with a regular one.

Reply:
I didn't dive deep into the causes... but it worked both in tests and UI :sweat_smile: 

Changed anyway :)